packages/sdks/snippets/angular/src/app/product-details/product-details.component.ts

```diff
@@ -0,0 +1,47 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import {
+  Content,
```

Comment:
we dont use `Content` in this example, so you can remove this import

Reply:
Ah, right! updated.